.changeset/orange-cobras-shave.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-jenkins-backend': minor
```

Comment:
Not a breaking change in either of the existing packages right? Should be `patch` in that case